index.html

```diff
@@ -1512,113 +1499,146 @@ <h1>WoT Architecture</h1>
-                        a thing</em>.</li>
-                <li><a>WoT Server</a>: A thing can behave as a
-                    server, which is a component that reacts to requests
-                    from other things. Requests query or modify the
-                    state of a thing, we call this interaction <em>to
-                        expose a thing</em>. Note that the server, is not
-                    purely a passive reactive component, since it can
-                    offer a subscribe/notify mechanism, which allows
-                    clients to be notified about state changes of a
-                    thing.</li>
-                <li><a>WoT Servient</a>: A thing can behave both
-                    as a client and a server, i.e. it can <em>expose</em>
-                    functionality to other things and it can <em>consume</em>
-                    functionalities that are provided by other things.</li>
-            </ul>
-
-            <p>Note that these concepts are logical definitions
-                based on the tasks of a thing. They do not require that
-                an exposed thing must be implemented with a (Web) server
-                and a consumed thing as a (Web) client.</p>
+                A <a>Thing</a> is the abstraction of a physical or virtual entity (e.g., a device or a room) and is described by standardized metadata.
+                In W3C WoT, the description metadata MUST be a WoT Thing Description (TD) [[!wot-thing-description]].
+                <a>Consumers</a> MUST be able to process the <a>WoT Thing Description</a> format, which is based on JSON [[!RFC8259]].
+                The format can be processed either through classic JSON libraries or a JSON-LD processor, as the underlying information model is graph-based and its serialization compatible with JSON-LD 1.1 [[?json-ld-syntax]].
+                A <a>TD</a> is instance-specific (i.e., describes an individual Thing, not types of Things)
+                and is the default Web representation of a <a>Thing</a>.
+                There MAY be other Web representations of a <a>Thing</a> such as an HTML-based user interface or simply an image of the physical entity.
+                However, the <a>WoT Thing Description</a> is a standardized, machine-understandable representation format
+                that allows <a>Consumers</a> to discover and interpret the capabilities of a Thing (through semantic annotations)
+                and to adapt to different implementations (e.g., different protocols or data structures) when interacting with a Thing,
+                thereby enabling interoperability across different <a>IoT platforms</a>, i.e., different ecosystems and standards.
+            </p>
+            <p class="ednote" title="TODO">
+                Simple figure to illustrate Consumer--Thing interaction.
+            </p>
+            <p>
+                A <a>Thing</a> can also be the abstraction of a virtual entity,
```

Comment:
Is a thing an abstraction of a virtual entity or is the virtual entity already the abstraction?

Reply:
The virtual entity is the composition. The composition can be treated under the same Thing abstraction as physical entities.

Reply:
Broke apart sentence to be clear.